src/classes/CRLP_Batch_Base.cls

```diff
@@ -306,50 +199,65 @@ public abstract class CRLP_Batch_Base {
      * @return List<Partial_Soft_Credit__c>
      */
     protected List<Partial_Soft_Credit__c> buildPartialSoftCreditRecordsFromOCRs(List<OpportunityContactRole> detailRecords) {
+        // These are both referenced in the SOQL created by buildDetailObjectQueryForRollup().
```

Comment:
I added the comment because IntelliJ is smart enough to recognize that the "contactIds" var is never referenced in the code here. However, that's because it's added to the SOQL statement created by `buildDetailObjectQueryForRollup`. So, when I removed the _unused_ var the first time it ended up throwing an exception and took me a few minutes to track down.

Overall I really want to finish rebuilding `CRLP_Query_SEL`, which would probably help here, but time is starting to run out on the refactoring.